kymatio/scattering1d/frontend/torch_frontend.py

```diff
@@ -27,27 +28,41 @@ def register_filters(self):
         saves those arrays as module's buffers."""
         n = 0
         # prepare for pytorch
+        # TODO: have option for different shaped filter depending on
+        # convolution primitive?
         for k in self.phi_f.keys():
             if type(k) != str:
                 # view(-1, 1).repeat(1, 2) because real numbers!
-                self.phi_f[k] = torch.from_numpy(
-                    self.phi_f[k]).float().view(-1, 1)
+                if self.convolution == 'spatial':
```

Comment:
What's going on here?

Reply:
This is code I added to make things work for the spatial convolution.  

I should remove it seeing as we are not adding a spatial convolution.

Reply:
Ok. Yeah remove it for now.